src/vs/workbench/parts/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -908,29 +919,35 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 		return Object.keys(result);
 	}
 
-	ignoreExtensionRecommendation(extensionId: string): void {
-		/* __GDPR__
-			"extensionsRecommendations:ignoreRecommendation" : {
-				"recommendationReason": { "classification": "SystemMetaData", "purpose": "FeatureInsight", "isMeasurement": true },
-				"extensionId": { "classification": "PublicNonPersonalData", "purpose": "FeatureInsight" }
+	toggleExtensionRecommendation(extensionId: string, shouldRecommend: boolean): void {
+		if (shouldRecommend) {
+			this._globallyIgnoredRecommendations = distinct(<string[]>JSON.parse(this.storageService.get('extensionsAssistant/ignored_recommendations', StorageScope.GLOBAL, '[]')))
```

Comment:
I dont believe we need to read from the storage here. `_globallyIgnoredRecommendations` will have all the ignored recommendations